### PR TITLE
Made chip_select default to None

### LINKED ISSUE #8: Need a way to create an SPI device without a chip select pin
e.g. `spi_device = SPIDevice(busio.SPI(board.SCK, board.MOSI, board.MISO), None, baudrate=4000000)`

This is for things like displays that have other signals to be managed during a transaction. We want to tell the busDevice to not worry about chip select, we will manage it ourselves manually.

### PR DESCRIPTION
**Pull request by cerickson**
Resolves #8. I didn't have hardware to fully test at PyCon, please test with BMP280 with/without specifying chip_select.

### CODE REVIEW

**Comment by cerickson** [comment]
Tested OK with a feather M0 connecting MISO directly to MOSI.

**Comment by dhalbert** [comment]
pylint complaining about trailing whitespace:
```
Using config file /home/travis/build/adafruit/Adafruit_CircuitPython_BusDevice/.pylintrc
************* Module adafruit_bus_device.spi_device
C: 68, 0: Trailing whitespace (trailing-whitespace)
-----------------------------------
Your code has been rated at 9.82/10
```

File: adafruit_bus_device/spi_device.py [hunk 1]
```diff
@@ -65,7 +65,8 @@ class SPIDevice:
             with device as spi:
                 spi.write(bytes_read)
     """
-    def __init__(self, spi, chip_select=None, *, baudrate=100000, polarity=0, phase=0, extra_clocks=0):
+    def __init__(self, spi, chip_select=None, *,
```
Comment (dhalbert) [inline adafruit_bus_device/spi_device.py]: remove trailing space on this line

**Review by dhalbert — CHANGES_REQUESTED** [review]

**Review by dhalbert — APPROVED** [review]
Great! Thank you!!

**Pull request merged**